Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -4107,6 +4107,16 @@ - (BOOL)canPerformActionForWebView:(SEL)action withSender:(id)sender
 
     if (action == @selector(findAndReplace:))
         return self.webView._findInteractionEnabled && self.supportsTextReplacement;
+
+    if (action == @selector(useSelectionForFind:) || action == @selector(_findSelected:)) {
+        if (!self.webView._findInteractionEnabled)
+            return NO;
+
+        if (!self.selectedText.length)
```

Comment:
Nit - `selectedText` is a bit of a misnomer here, since it can return a non-empty string even in the case of a caret selection in editable content (if the caret is next to a word).

Perhaps it would be more appropriate to check `_page->editorState().selectionIsRange` as well?

Reply:
Yes! Updated.